src/middleware.coffee

```diff
@@ -21,6 +21,7 @@ class Middleware
   # Returns nothing
   # Returns before executing any middleware
   execute: (context, next, done) ->
+    done ||= ->
```

Comment:
I don't think so, but this might be the extent of my coffeescript knowledge. `?=` would let us pass in `0` or 100 other falsey things and have them be silently overwritten. `||=` would only fire in case of a skipped argument.